snmp_american_power_conversion/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "6b5325b8-443d-42e0-8545-f7dc42acacb4",
+  "app_id": "snmp-american-power-conversion",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect SNMP metrics from your American Power Conversion network devices.",
+    "title": "American Power Conversion",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Monitoring",
+      "Category::Notification",
+      "Category::Network",
+      "Category::Autodiscovery",
+      "Category::SNMP"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "American Power Conversion",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "snmp_american_power_conversion.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
```

Comment:
```suggestion
```
We don't provide a metadata.csv (it's part of the snmp integration), and the prefix isn't the one set up here.
Not sure we can have a `metrics` section which makes sense here, looks like other snmp tiles don't define one [[1](https://github.com/DataDog/integrations-core/blob/master/snmp_dell/manifest.json)] [[2](https://github.com/DataDog/integrations-core/blob/master/snmp_f5/manifest.json)]

Reply:
Nice catch, this was generated by the `ddev create` tool. We ought to fix that for the future right?  In any case, I can remove them manually